lib/ansible/plugins/filter/core.py

```diff
@@ -83,11 +83,18 @@ def to_json(a, profile: str | None = None, vault_to_text: t.Any = ..., preproces
     return json.dumps(a, cls=cls, **kwargs)
 
 
-def to_nice_json(a, indent=4, sort_keys=True, **kwargs):
-    """Make verbose, human-readable JSON."""
-    # TODO separators can be potentially exposed to the user as well
-    kwargs.pop('separators', None)
-    return to_json(a, indent=indent, sort_keys=sort_keys, separators=(',', ': '), **kwargs)
+def to_nice_json(a, indent=4, sort_keys=True, separators=None, **kwargs):
```

Comment:
Since the function signature is being updated, it should include type annotations.